test/test_runner.sh

```diff
@@ -27,10 +27,13 @@ function execute() {
 }
 
 # Tests to run. Default is "." (i.e. the current directory).
-TESTS=("${@:-.}")
+TESTS_ROOT=("${@:-.}")
+# Serial tests
+SERIAL_TESTS=("${TESTS_ROOT[@]}/serial")
 
-# The number of parallel jobs to execute
+# The number of parallel jobs to execute non-serial tests
 export JOBS=${JOBS:-$(($(nproc --all) * 4))}
 
 # Run the tests.
-execute bats --jobs "$JOBS" --tap "${TESTS[@]}"
+execute bats --jobs "$JOBS" --tap "${TESTS_ROOT[@]}" 
```

Comment:
execute replaces the current binary with bats so we can't use bash syntax to combine with other invocation or split this out to the next line. Instead, can we just invoke bats without the exec?

Reply:
Fixed

Reply:
@mrunalp I believe `execute` was just a function we defined to run time and put the stderr into stdout, I don't think we need to drop it (though having two separate executes for the two  lines seems like a good idea)